Compiler warning member field initialization order

Compiler warning:
```
src/Button.cpp:11:6: warning: field 'mMouseHover' will be initialized after field 'mImage' [-Wreorder]
                                        mMouseHover(false),
                                        ^
```

The constructor initializer list does not match the member variable declaration order. Initialization order will be the member variable declaration order, not what is shown in the constructor initializer list.

--- a/src/Button.cpp
+++ b/src/Button.cpp
@@ -8,8 +8,8 @@ using namespace NAS2D;
 
 Button::Button():	mState(STATE_NORMAL),
 					mType(BUTTON_NORMAL),
-					mMouseHover(false),
-					mImage(nullptr)
+					mImage(nullptr),
+					mMouseHover(false)
 {
 	Utility<EventHandler>::get().mouseButtonDown().connect(this, &Button::onMouseDown);
 	Utility<EventHandler>::get().mouseButtonUp().connect(this, &Button::onMouseUp);

